examples/misc/raptor.py

```diff
@@ -119,8 +119,8 @@ def task_state_cb(task, state):
                         'action': rp.TRANSFER})
         pilot.prepare_env(env_name='ve_raptor',
                           env_spec={'type'   : 'venv',
-                                    'version': '3.8',
-                                    'path'   : '$RP_RESOURCE_SANDBOX/ve_raptor/',
+                                    'version': '3.6',
+                                    'path'   : '/tmp/ve_raptor/',
```

Comment:
`/tmp/` is only accessible from within the compute node itself. It might problematic in the future if `raptor_script` for some reason needed to use the `ve_raptor` from another node,  otherwise, this looks fine for me.

Reply:
Correct - but since we need to specify *some* path, the example won't be easily portable out of the box anyway... :-/